docs/blog/pydantic-v2-alpha.md

```diff
@@ -178,4 +178,22 @@ The following config settings have been renamed:
 
 ### AnalyzedType
 
-TODO we need to give a brief into to this and how to use it.
+In Pydantic V1 if you wanted to validate a non-`BaseModel` type (like `Dict[str, int]` or `List[SomeModel]`) you had to use a "root" model.
```

Comment:
There was also `tools.parse_obj_as`, which was added specifically for this purpose, and had a pretty clean API (imo). The important reasons to drop it were:
* It was inefficient — building the parsing model as part of the call (though admittedly it had an lru_cache with cache size 2048, so maybe not super likely to hit that).
* It wasn't super compatible with our new approach to serialization.

I guess you could argue that in some sense, the `pydantic.tools._get_parsing_type` has been replaced by `AnalyzedType` which does all of that and more. But considering that's more of an implementation detail I don't think that needs to be called out to users.

I _do_ think we should mention `pydantic.tools.parse_obj_as` here though.

Reply:
(I'll also note that `pydantic.tools` had a function for getting the json schema of arbitrary types as well, so it really seems to me that the main benefit over the old approach is the serialization stuff, and any possible lru_cache-induced issues)

Reply:
To that end, might make sense to say something about how we had utility functions for implementing this workflow for `__root__` models before, and because it is now simplified we have deprecated those functions. And you can replace `parse_obj_as(Dict[int, List[str], data)` with `AnalyzedType(Dict[int, List[str]]).validate_python(data)`, etc.

Reply:
Do we need to explain why we're replacing `parse_obj_as` with this? Or can I just say "this replaces `pydantic.tools.parse_obj_as` and `pydantic.tools.json_schema` (or whatever the latter was called)

Reply:
I think we're keeping those types, just deprecated, and offering a 1 to 1 replacement, so that's fine.

The other big changes is that there's no `BaseModel` in the background, so performance should increased markedly.